### PR TITLE
Bugfix/fix validation

### PR DESCRIPTION
**Pull request by nmenezes0**
Fix validation on the page where users are recording learning.

If users leave either the hours field blank or the minutes field blank (not both) - there should be no error, and we assume that it's zero.

If both the hours and the minutes are blank, then there should be an error.

### CODE REVIEW

File: one_big_thing/learning/schemas.py [hunk 1]
```diff
@@ -90,13 +101,13 @@ def validate_time_to_complete(value):
...
-    general_error = "Please enter the hours this course took to complete, for example, 2"
+    general_error = HOURS_ERROR
     max_hours_error = f"The course should be less than {constants.HOURS_LIMIT} hours"
     validate_positive_integer(value, max=constants.HOURS_LIMIT, error_msg=general_error, error_msg_max=max_hours_error)
 
 
 def validate_time_to_complete_minutes(value):
-    minutes_error = "Please enter the minutes this course took to complete, between 0 and 59"
+    minutes_error = MINUTES_ERROR
     validate_positive_integer(value, max=59, error_msg=minutes_error)
```
Comment (252afh) [inline one_big_thing/learning/schemas.py:111]: Not sure why, but entering a large minute causes the error to be returning twice
image

Comment (252afh) [reply one_big_thing/learning/schemas.py:111]: Same with hours 
image

Comment (252afh) [reply one_big_thing/learning/schemas.py:111]: Same with large numbers
image

**Review by 252afh — CHANGES_REQUESTED** [review]

**Comment by nmenezes0** [comment]
Ready for you to break it again @252afh - think I've fixed the issues with duplicate messages.

**Review by 252afh — APPROVED** [review]

**Pull request merged**